list_map.py

```diff
@@ -0,0 +1,178 @@
+from cartesian_coordinate import CartesianCoordinate
+from mapping_interface import MapInterface
+from mapping_interface import PointState
+
+
+class ListMap(MapInterface):
+    """
+    @brief Class ListMap provides functionality to create a map using keys and values. 
+    """
+
+    def __init__(self):
+        """
+        @brief ListMap constructor used to create a basic list map.
+        """
+
+        self.map = [
```

Comment:
Why does this map get initialized to an arbitrary size of 3x3 while the `KeyValueMap` stays empty? I'd suggest using at most a single `PointState`, and set all three variables (`origin`, `top_left_point`, `bot_right_point`) to the coordinates `0,0`.